src/frontend/apps/lti_site/components/LocalizedTimedTextTrackUpload/LanguageSelect/index.spec.tsx

```diff
@@ -20,18 +19,26 @@ jest.mock('lib-components', () => ({
 const onChangeMock = jest.fn();
 
 const languageChoices = [
-  { label: 'English', value: 'en' },
-  { label: 'French', value: 'fr' },
-  { label: 'Spanish', value: 'es' },
-  { label: 'Slovenian', value: 'sl' },
-  { label: 'Swedish', value: 'sv' },
+  { display_name: 'English', value: 'en' },
+  { display_name: 'French', value: 'fr' },
+  { display_name: 'Spanish', value: 'es' },
+  { display_name: 'Slovenian', value: 'sl' },
+  { display_name: 'Swedish', value: 'sv' },
 ];
 
 describe('<LanguageSelect />', () => {
-  it('renders the component with instructor local language available', () => {
-    useTimedTextTrackLanguageChoices.setState({
-      choices: languageChoices,
-    });
+  afterEach(() => {
+    fetchMock.restore();
+  });
+
+  it('renders the component with instructor local language available', async () => {
+    fetchMock.mock(
```

Comment:
If the mock is in all the tests, considere to add it in the beforeEach methods.

Reply:
No, there is one test where I mock wiht a 500 error